README.md

```diff
@@ -10,15 +10,15 @@ Note that although petastorm depends on opencv, it is defined as an extra packag
 to be installed by debian package (or source) as opposed to pip. It is recommended to install it when
 testing out petastorm however (and is therefore shown in the examples).  
 
-Install with tensorflow CPU dependency:
-```bash
-pip install petastorm[opencv, tf]
-```
+Please follow the following instructions to develop Petastorm:
```

Comment:
The instructions that we had here were for petastorm users, not developers (installing off pypi server). I think previous lines should still be here. Suggest we move developer related instructions to a separate `docs/development.md` (this is not a common case). 

It can be more convenient when all four lines are in the same block, so you can copy paste it at once to the terminal. We can show a safer tensorflow-cpu version and have a side comment about tf_gpu.